controllers/employeeDashboard.controller.js

```diff
@@ -0,0 +1,61 @@
+const User = require("./../modules/user.module");
+const {
+    setHomeAddress,
+    setVacationRequest,
```

Comment:
vacation request is not this controller functionality to handle. it's vacation.DAL'a job to handle this